maat-court-data-api/src/main/java/gov/uk/courtdata/repository/RepOrderMvoRegRepository.java

```diff
@@ -0,0 +1,14 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.entity.RepOrderMvoRegEntity;
+import gov.uk.courtdata.reporder.projection.RepOrderMvoRegEntityInfo;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface RepOrderMvoRegRepository extends JpaRepository<RepOrderMvoRegEntity, Integer>, JpaSpecificationExecutor<RepOrderMvoRegEntity> {
+    List<RepOrderMvoRegEntityInfo> findByMvo_IdAndDateDeletedIsNull(Integer mvoId);
```

Comment:
You should be able to drop the underscore from the method name, it makes it more difficult to read.

Reply:
Will look into this. But the DB field name is named as MVO_ID and hence it was auto generated as above.